Source/Renderer/Context.js

```diff
@@ -758,6 +758,7 @@ define([
     });
 
     function validateFramebuffer(context, framebuffer) {
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
In Context.js, can you add a doc comment that `validateFramebuffer` only works in debug builds?  It is an internal API and so that is no problem.